test/kubemark/start-kubemark.sh

```diff
@@ -467,7 +472,11 @@ start-master-components
 # Setup for hollow-nodes.
 echo ""
 echo -e "${color_yellow}STARTING SETUP FOR HOLLOW-NODES${color_norm}"
-create-and-upload-hollow-node-image
+if [[ "${KUBEMARK_BAZEL_BUILD:-}" =~ ^[yY]$ ]]; then
```

Comment:
I don't think this change should be made. Seems like a bad rebase - please revert this part.